.github/workflows/build.yml

```diff
@@ -0,0 +1,23 @@
+name: Build
+
+on: 
+  pull_request:
+      branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    env:
+      GO11MODULE: 'on'
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: true
+          fetch-depth: 0 # we want all tags for version check.
+          lfs: true
+      - uses: actions/setup-go@v2
+        with:
+          go-version: '^1.11.0'
```

Comment:
Lets use something more recent.  Maybe 1.14 or 1.15.  I think GO11MODULE isn't needed after that.

Reply:
This is matching what we had in Travis-CI.  I'm not going to bump to 1.14/1.15 unless we're officially dropping support for older Go version here, which we're not.  This is just a migration.